src/bundle/Resources/views/themes/admin/account/forgot_password/index.html.twig

```diff
@@ -3,7 +3,10 @@
 {% form_theme form_forgot_user_password '@ezdesign/account/form_fields.html.twig'  %}
 
 {%- block content -%}
-    <h2 class="ez-login__header">{{ 'authentication.reset_your_password'|trans|desc('Reset your password') }}</h2>
+    <h2 class="ez-login__actions-headline">{{ 'authentication.reset_your_password'|trans|desc('Reset your password') }}</h2>
+    {% if reason is defined and reason == userForgotPasswordReasonMigration %}
+        <p>{{ 'ezplatform.forgot_password.reset_your_password.reason.migration'|trans|desc('Your password has expired, change it!') }}</p>
```

Comment:
Is this key already present in Xliff?
@ezsystems/documentation-team needs to look at the translated text ;)

Reply:
https://github.com/ezsystems/ezplatform-user/pull/75/files#diff-e3a57e1770eea106e58e94d5d485009cR41

But yes, I think we should add it to _local_ xliff as well

Reply:
added

Reply:
> added

Ok, so what's left is user message to be checked by @ezsystems/documentation-team. The same thing applies to ezsystems/ezplatform-user#75 probably.